include/boost/geometry/policies/robustness/segment_ratio.hpp

```diff
@@ -335,12 +342,17 @@ private :
     // Boost.Rational is used if the approximations are close.
     // Reason: performance, Boost.Rational does a GCD by default and also the
     // comparisons contain while-loops.
-    fp_type m_approximation;
+    floating_point_type m_approximation;
 
+    inline bool close_to(thistype const& other) const
+    {
+        return geometry::math::abs(m_approximation - other.m_approximation)
+                < 50;
```

Comment:
Is 50 here a magic number or is it related with some values declared in other parts of the file?

Reply:
It's for efficiency only, and related to the scale value 1000000.
If it is close_to, then `operator==` uses `geometry::math::equals`, otherwise it's just false.

This was already the case, the method is just moved.